apc/proxy.php

```diff
@@ -9,12 +9,35 @@
 	}
 }
 
+/**
+ * Determines if SSL is used.
+ *
+ * @see is_ssl() (wp-includes/load.php).
+ *
+ * @return bool True if SSL, otherwise false.
+ */
+function cachify_is_ssl() {
+	if ( isset( $_SERVER['HTTPS'] ) ) {
+		if ( 'on' == strtolower( $_SERVER['HTTPS'] ) ) {
```

Comment:
We should check strict, like `===` to create a solid code base. Would you change this, enhance this to more scrict source?